README.md

```diff
@@ -44,4 +44,9 @@ An extension to help you in editing .arb files, used for localization of applica
 Escaping quotes can be turned off by including a `l10n.yaml` file with the line:
 ```yaml
 use-escaping: false
+```
+
+To set a reference file, set `"@@x-reference": true` element in your `arb` file, or set a file to be the reference in the `l10n.yaml` using:
```

Comment:
What does setting this to `true` mean? Where does the template/reference file come from if the value is only `true`?

Reply:
Yeah it makes more sense for this also to be a file path to the template file, or empty if this is a template itself. And to use one of the two words consistently :D